File: ThemisBlazorApp2.0/ThemisBlazorApp2.0/Data/DataTransferObjects/UserStory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ThemisBlazorApp2._0.Data.DataTransferObjects
{
    public class UserStory
    {
        public String Description { get; set; }
        public UserStory(string description)
        {
            this.Description = description;
        }
    }
}

File: Logic_API/src/main/java/dataTransferObjects/UserStory.java
package dataTransferObjects;

public class UserStory
{
    private String Description;

    public UserStory(String description)
    {
        this.Description = description;
    }

    public String getDescription()
    {
        return Description;
    }

    public void setDescription(String description)
    {
        this.Description = description;
    }
}
